src/main/resources/templates/login.html

```diff
@@ -19,16 +19,7 @@
             integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
             crossorigin="anonymous"></script>
     <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
-    <script>
-        $(document).ready(function() {
-            $("#nav-placeholder").load("/navbar");
-        });
-    </script>
-    <script>
-        $(document).ready(function() {
-            $("#footer-placeholder").load("/footer");
-        });
-    </script>
+    <script src="/js/navbarFooter.js" type="text/javascript"></script>
```

Comment:
nice that you did this everywhere, but it was absolutely not part of the ticket. next time, make your own branch and own mr for this